devchat.go

```diff
@@ -321,11 +321,22 @@ func (u *user) rWriteln(msg string) {
 	}
 }
 
+
 func (u *user) pickUsername(possibleName string) (ok bool) {
 	possibleName = cleanName(possibleName)
 	var err error
-	for possibleName == "" || possibleName == "devbot" || strings.HasPrefix(possibleName, "#") || userDuplicate(u.room, possibleName) {
-		u.writeln("", "Your username is already in use. Pick a different one:")
+	for {
+		if possibleName == "" {
```

Comment:
This case is used elsewhere to allow prompting for a name right away, please remove the u.writeln() call beneath it